R/download_package.R

```diff
@@ -0,0 +1,327 @@
+#' Remove and substitute special characters in a dataOne identifier
+#'
+#' This is a helper function for the 'download_package' function. Remove special
+#' characters from a package identifier for use as a filename prefix. Can be
+#' generalized for use with any pid.
+#'
```

Comment:
I was a bit confused by this.  Perhaps be a bit more explicit and rephrase "for use as a filename prefix" to "to use in a filename, which does not allow special characters"